x-pack/legacy/plugins/reporting/export_types/common/execute_job/decrypt_job_headers.ts

```diff
@@ -6,26 +6,27 @@
 
 import { i18n } from '@kbn/i18n';
 import { cryptoFactory } from '../../../server/lib/crypto';
-import { CryptoFactory, ServerFacade, Logger } from '../../../types';
+import { ReportingConfig } from '../../../server/types';
+import { CryptoFactory, Logger } from '../../../types';
```

Comment:
In a future PR, we should clean up the type declarations. Most of the ones in the original `types` file should probably be in `server/types` at this point. That should simplify imports quite a bit

Reply:
++ been liking that pattern as well